src/demo/app-routing.module.ts

```diff
@@ -20,6 +20,7 @@ import { SearchHighlightExampleComponent }
 import { SortExampleComponent } from '../app/sort/examples/sort-example.component';
 import { SortArrayExampleComponent } from '../app/pipe/sort-array/examples/sort-array-example.component';
 import { SparklineExampleComponent } from '../app/chart/sparkline/examples/sparkline-example.component';
+import { DonutExampleComponent } from '../app/chart/donut/examples/donut-example.component';
```

Comment:
Alphabetical order please

Reply:
Okay will do.

Thoughts on adding `ordered-imports` and the like to enforce tslinting of alphabetical order as a separate issue/PR?  Also I noticed `tslint` was updated from 4.x to 5.x and the tslint config file contains unsupported configuration for `member-ordering`. It doesn't seem to break anything but I think a fix would be good for those using editors that read from the config file to show tslint warnings.

Reply:
Yes, it would be good to fix this. A separate PR would probably be best.

Reply:
The member-ordering seems to be working; for example, if I add a private method before a public one. Although, it's possible that config could be deprecated?

Unfortunately, none of that enforces import ordering, just fields, methods, and constructors. However, I did find adding this does the trick:

"ordered-imports": true,